compiler/tests/evm_contracts.rs

```diff
@@ -124,7 +123,7 @@ impl ContractHarness {
                 })
                 .collect::<Vec<_>>();
 
-            if !outputs_for_event.contains(&expected_output) {
+            if !outputs_for_event.iter().any(|v| &v == expected_output) {
```

Comment:
No big deal but we generally try to avoid single letter variables. This could be named `val` instead.